Grooming/Grooming.md

```diff
@@ -2,42 +2,48 @@
 ```bash
 {
 "**Prompt:**" "How can I schedule a grooming appointment for my pet through Petofy?"
+"**Response:**": "To schedule a grooming session visit https://www.petofy.com/. Now follow these steps:-
+ - Go to Grooming under Find services
+ - Select your city from the list.
+ - Select the groomer who meets your criteria
+Make sure are logged into your Petofy account for booking the groomer."
+ 
 "**Response:**" "You can simply Log in to your Petofy account, add your location, go to the grooming section, choose your preferred groomer, and select a convenient appointment slot."
 }
 ```
 ```bash
 {
 "**Prompt:**" "What qualifications and experience do the groomers on Petofy have?"
-"**Response:**" "Petofy partners with highly qualified and experienced groomers. You can check out groomers' profiles at https://www.petofy.com/dehradun/pet-grooming to learn more about their qualifications and experience."
+"**Response:**" "Petofy partners with highly qualified and experienced groomers. You can check out groomers' profiles at https://www.petofy.com/dehradun/pet-grooming to learn more about their qualifications and experience." (Must talk of policy. The link is non applicable here).
```

Comment:
- Give a link to policy if available (Ask Petofy team).
- Don't redirect to Dehradun link.

Reply:
done